src/components/SimpleSelect.js

```diff
@@ -48,33 +50,38 @@ const SimpleSelect = React.createClass({
       this.props.onScrimClick(e);
     }
 
-    item.onClick(e);
+    item.onClick(e, item);
   },
 
   render () {
     const styles = this.styles();
 
     return (
       <div style={styles.component}>
-        <div style={styles.menu}>
+        <Listbox
+          aria-label={this.props['aria-label']}
```

Comment:
let me know what your thinking on the `aria-label` vs `ariaLabel` is. i mentioned internally i prefer the ariaLabel but I could be convinced if you have strong feelings towards the other